elements/lisk-transactions/src/12_multisignature_transaction.ts

```diff
@@ -376,10 +376,10 @@ export class MultisignatureTransaction extends BaseTransaction {
 	public signAll(
 		networkIdentifier: string,
 		senderPassphrase: string,
+		passphrases?: ReadonlyArray<string>,
```

Comment:
shouldn't `senderPassphrase` also be optional if it has the same interface?

Reply:
Multisig registration is the exception I think as we need the sender to be in position `0` of the signatures array so this works as a kind of obligation for the account being transformed to be the first one to sign and share the transaction to other members